Sources/ClientRuntime/Networking/Http/HttpContext.swift

```diff
@@ -72,6 +84,9 @@ public class HttpContextBuilder {
     let hostPrefix = AttributeKey<String>(name: "HostPrefix")
     let logger = AttributeKey<LogAgent>(name: "Logger")
     let partitionID = AttributeKey<String>(name: "PartitionID")
+    public static let messageEncoder = AttributeKey<MessageEncoder>(name: "MessageEncoder")
+    public static let messageDecoder = AttributeKey<MessageDecoder>(name: "MessageDecoder")
+    public static let messageSigner = AttributeKey<MessageSigner>(name: "MessageSigner")
```

Comment:
Do these need to be public given the values are accessed through functions on HttpContext?

Additionally, it'd be great if you could also update the existing keys to static and reuse them in the Context as well. While not part of event streams, it should be low-effort and would make everything consistent.

Reply:
they are access across packages. So yes, we need them public.